source/agora/utils/PrettyPrinter.d

```diff
@@ -355,7 +358,9 @@ private struct BlockHeaderFmt
 { utxo: 0xb20d...08eb, seed: 0xa050...2cb4, cycles: 20, sig: 0x052e...6b31 }
 { utxo: 0xdb39...2d85, seed: 0xdd1b...7bfa, cycles: 20, sig: 0x0e00...4fe2 }]
 Signature: 0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000,
-Validators: [0]`;
+Validators: [0],
+RandomSeed: [0x0000...0000],
+MPV: []`;
```

Comment:
In the future I think we should name this just `MV` that stands for "missing validators".

Reply:
> In the future I think we should name this just MV that stands for "missing validators".

I agree with it.